capabilityreceiver/pkg/capabilityhandler/capabilityreceiver_integration_test.go

```diff
@@ -296,7 +296,7 @@ func Test_saveMsgInDB(t *testing.T) {
 	var http_200_ct int
```

Comment:
Could update on lines 83 & 90 to add `RequestedFhirVersion` to `endpointMetadataDefaultRequest` and `endpointMetadataRequestVersion1`. Could also add a check in `Test_saveMsgInDB` where adding an endpoint with the same URL but a different requested version would add a new entry to the availability table.